code/datums/components/surgery_initiator.dm

```diff
@@ -69,7 +69,7 @@
 					continue
 			else if(carbontarget && surgeryloop_two.requires_bodypart) //mob with no limb in surgery zone when we need a limb
 				continue
-			if(surgeryloop_two.lying_required && (livingtarget.mobility_flags & MOBILITY_STAND))
+			if(surgeryloop_two.lying_required && livingtarget.body_position != LYING_DOWN)
```

Comment:
```suggestion
			if(surgeryloop_two.lying_required && livingtarget.body_position == STANDING_UP)
```

Reply:
This is the same as before but less safe if more body positions are added.